### ISSUE TITLE
BiQubitsMixer also can't decompose

### ISSUE THREAD

**Issue by fdmalone** [quantumlib/Qualtran#389]
``` python
import cirq_ft
from qualtran.cirq_interop import CirqGateAsBloq
from qualtran.drawing import show_bloq

bloq = CirqGateAsBloq(cirq_ft.algos.BiQubitsMixer(False))
show_bloq(bloq.decompose_bloq())
```
BloqError: During finalization, {Soquet(binst=BloqInstance(bloq=Split(n=3), i=0), reg=Register(name='split', bitsize=1, shape=(3,), side=<Side.RIGHT: 2>), idx=(2,)), Soquet(binst=BloqInstance(bloq=Split(n=3), i=0), reg=Register(name='split', bitsize=1, shape=(3,), side=<Side.RIGHT: 2>), idx=(1,))} Soquets were not used.

**Comment by fdmalone** [comment]
similar to #388

**Comment by fdmalone** [comment]
I'll push a fix

**Comment by fdmalone** [comment]
Following discussion in quantumlib/Cirq#6613 I'm leaving this issue open to track the need for a better docstring / renaming of ancilla for the BiQubitsMixed cirq_ft gate which is used for the comparators.

**Comment by mpharrigan** [comment]
did you mean to close this?

**Comment by fdmalone** [comment]
Is there some auto close issue feature. Anyway it's fixed following #391
